<?xml version="1.0"?>
<launch>

  <arg name="node_name"     default="node_01" />

  <!--*************************************************************-->
  <!-- Hi-Ros Skeleton Optimizer Node: Parameters override and Run -->
  <!--*************************************************************-->
  <include file="$(find hiros_skeleton_optimizer)/launch/default.launch">
    <arg name="node_required"                            value="true"             />
    <arg name="node_name"                                value="$(arg node_name)" />

    <arg name="input_topic_name"                         value="topic"            />
    <arg name="output_topic_name"                        value="skeleton_group"   />

    <arg name="number_of_frames_for_calibration"         value="50"               />
    <arg name="max_calibration_coefficient_of_variation" value="0.1"              />
    <arg name="outlier_threshold"                        value="0.2"              />

    <arg name="export_calibration"                       value="true"             />
    <arg name="calibration_file"                         value="$(find hiros_skeleton_optimizer)/config/test_calib.yaml" />
  </include>

</launch>
